packages/e2e-tests/specs/widgets/customizing-widgets.test.js

```diff
@@ -18,6 +18,10 @@ import {
 // eslint-disable-next-line no-restricted-imports
 import { find } from 'puppeteer-testing-library';
 
+const twentyTwentyError = `Stylesheet twentytwenty-block-editor-styles-css was not properly added.
+For blocks, use the block API's style (https://developer.wordpress.org/block-editor/reference-guides/block-api/block-metadata/#style) or editorStyle (https://developer.wordpress.org/block-editor/reference-guides/block-api/block-metadata/#editor-style).
+For themes, use add_editor_style (https://developer.wordpress.org/block-editor/how-to-guides/themes/theme-support/#editor-styles).`;
```

Comment:
The 2020 theme is not adding editor styles through `add_editor_style`. For some reason the widgets tests are using this theme. It's outside the scope of this PR to fix this.